distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -275,10 +274,121 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Maven is now responsible for starting and stopping the Solr server for itests.

Reply:
There shouldn't be any issue, but if something does arise, we can cover that in a follow-up improvement.